flask_and_kafka/consumer.py

```diff
@@ -22,12 +23,16 @@ def __init__(self, app: Flask = None) -> None:
     def init_app(self, app: Flask) -> None:
         self._app = app
         app.extensions["kafka_consumer"] = self
-        self.consumer_logger = consumer_logger(
-            name="consumer_logger",
-            file=app.config.get(
-                "KAFKA_CONSUMER_LOG_PATH", "logs/kafka_consumer.log"
-            ),
-        )
+        logger_name = 'consumer_logger'
```

Comment:
I recommend having this part separated into a dedicated method to make testing and user interaction with it easier. Additionally, it would be excellent if we could have documentation for it 😃.